qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/command/CreateProcessInstanceTest.java

```diff
@@ -218,4 +237,41 @@ public void shouldRejectCreateBpmnProcessByNonExistingKey() {
         .isInstanceOf(ClientException.class)
         .hasMessageContaining("Expected to find process definition with key '123', but none found");
   }
+
+  @Test
+  public void shouldCreateWithStartInstructions() {
+    // when
+    final var instance =
+        CLIENT_RULE
+            .getClient()
+            .newCreateInstanceCommand()
+            .processDefinitionKey(thirdProcessDefinitionKey)
+            .startBeforeElement("end1")
+            .startBeforeElement("end2")
+            .send()
+            .join();
+
+    assertThat(instance.getProcessInstanceKey()).isPositive();
```

Comment:
:wrench: We could verify that the start instructions are really applied by using the `RecordingExporter`. For example, by checking that only the end events are activated. 

Currently, the test case would pass if the start instructions are ignored (in the client, the gateway, or the broker).

Reply:
I've opened #10382 for this